src/main/kotlin/org/opensearch/commons/notifications/NotificationsPluginInterface.kt

```diff
@@ -211,11 +210,6 @@ object NotificationsPluginInterface {
         request: LegacyPublishNotificationRequest,
         listener: ActionListener<LegacyPublishNotificationResponse>
     ) {
-        if (request.feature != FEATURE_INDEX_MANAGEMENT) {
```

Comment:
Might want to update the description (e.g. "Internal API only for the Index Management plugin.")